runner/docker/Dockerfile.live-base-comfyui

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=livepeer/comfyui-base:ai-runner-9166aa9
+ARG BASE_IMAGE=livepeer/comfyui-base:latest
```

Comment:
Can we maintain a fix image version here? We don't want this to update without us realizing, even if we know we can stay on the bleeding edge of `comfystream`. We'll always want to manually update this, test and only then deploy consciously to production. In the future we'd be able to automate most of this process as well. Ref: [[ai-runner devx phase 3]](https://discord.com/channels/423160867534929930/1332431145240887336/1342205704320323716)

Reply:
Updated to fixed version matching v0.0.5 tag which this PR has been tested with https://github.com/livepeer/comfystream/releases/tag/v0.0.5